test/e2e_node/runner/run_e2e.go

```diff
@@ -502,9 +502,11 @@ func getComputeClient() (*compute.Service, error) {
 }
 
 func deleteInstance(image string) {
-	_, err := computeService.Instances.Delete(*project, *zone, imageToInstanceName(image)).Do()
+	instanceName := imageToInstanceName(image)
```

Comment:
If it happens frequently, we should check. Otherwise, I think it's ok to fall back to the periodic cleanup task.